src/sentry/replays/usecases/ingest/dom_index.py

```diff
@@ -205,6 +205,16 @@ def get_user_actions(
             log["project_id"] = project_id
             log["replay_id"] = replay_id
             logger.info("SDK Options:", extra=log)
+        # log large dom mutation breadcrumb events
+        if (
+            event.get("type") == 5
+            and event.get("data", {}).get("tag") == "breadcrumb"
+            and event.get("data", {}).get("payload", {}).get("category") == "replay.mutations"
+        ):
+            log = event["data"].get("payload", {}).copy()
+            log["project_id"] = project_id
+            log["replay_id"] = replay_id
+            logger.info("Large DOM Mutations List:", extra=log)
```

Comment:
I'm thinking we should have a logging breadcrumb type so the SDK can send new payloads that are logged automatically.  Rather than us creating new log messages on a case by case basis.  We can have them auto-ingest them into bigquery.  It would prevent us from being a blocker to SDK experimentation.

Reply:
this is a great idea. cc @billyvg maybe for all the breadcrumbs we want to log going forward we could add an extra field to the breadcrumb that's like "log:True" or somesuch.

Reply:
Yeah this is a great idea!